Atspm/ConfigApi/Program.cs

```diff
@@ -81,12 +81,6 @@
     {
         o.GroupNameFormat = "'v'VVV";
         o.SubstituteApiVersionInUrl = true;
-
-        //configure query options(which cannot otherwise be configured by OData conventions)
-        //o.QueryOptions.Controller<JurisdictionController>()
```

Comment:
Add these back in. I kind of want them there for now so I don't forget how to do it.
Might changed some things over to this instead of using the config file